db/fixtures/miq_product_features.yml

```diff
@@ -3933,6 +3933,14 @@
         :description: Request to Provision VMs
         :feature_type: admin
         :identifier: vm_miq_request_new
+      - :name: Approve Provision VMs
+        :description: Approve Request to Provision VMs
+        :feature_type: admin
+        :identifier: vm_miq_request_approve
+      - :name: Deny Provision VMs
+        :description: Deny Request to Provision VMs
+        :feature_type: admin
+        :identifier: vm_miq_request_deny
```

Comment:
@dclarizio @gmcculloug this is exposing new approve/deny entitlement in the UI too, is this your expectation too to be able to approve/deny provisioning requests or should we be using different entitlements ?